meeting-notes/20181113.md

```diff
@@ -0,0 +1,203 @@
+     - membership heuristic sampling function should be synchronous: https://github.com/ipfs-shipyard/peer-star-app/pull/110
+     - chore: using aegir lint and fixes linting errors: https://github.com/ipfs-shipyard/peer-star-app/pull/116
+ - In progress:
+    - expose collaboration replication events: https://github.com/ipfs-shipyard/peer-star-app/pull/111
+ - Blocked:
+ - Next:
+    - continue on exposing collaboration replication events: https://github.com/ipfs-shipyard/peer-star-app/pull/111
+    - support different strategies for local store: https://github.com/ipfs-shipyard/peer-star-app/issues/117
+
+#### @satazor
+ - Concluded:
+   - working group 2019 roadmap, 1st iteration
+   - discussify:
+     - UI level QA: https://github.com/ipfs-shipyard/pm-discussify/issues/58
+     - Standardized styleguide comment names with @andreforsousa
+ - In progress:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - Next:
+   - Implement "show history" in the UI
+   - Add persistance of the data (pinner + sync UI language)
+     
+Note: Ask pedro about granularity of Discussify v1 milestones on the roadmap
+     
+ #### @andreforsousa    
+ - Concluded:
+    - Discussify:
+    - Style-guide updates with all scenarios and states (hover, focus, error feedback) for "comment-component"
+        - Comment
+        - Comment (ownership)
+        - Comment Actions
+        - Comment Edited
+        - Comment Edited History
+        - Comment Placeholder
+        - Comment Removed
+        - Comment Input (new reply)
+        - Comment Input (editing)
+    - QA & style-guide updates for the pre-alpha version
+        - FAB Shadow fix
+    - PeerPad UX/UI fixes:
+        - Grid width for 80columns accommodation (desktop)
+        - Dismissible notifications (desktop)
+        - Empty state illustration
+        - History revision
+        
+- Blocked:
+    - PeerPad mobile:
+    - Hi-Fi prototype to start looking for:
+        - Transitions between elements
+        - Start defining values for the eases
+        - This will be in the backlog until desktop version is properly defined and polished, following David’s feedback.
+    
+- In progress:
+  - Discussify Brand & Web Guidelines (manual)
+  - Look at new features for PeerPad (sync with Victor)
+
+ 
+#### @VictorBjelkholm
+ - Concluded:
+ 		- Error reporting on peerpad.net
+ - In Progress:
+ 		- UX review
+    - Pinner review
+ - Blocked:
+ - Next:
+    - Pinner deployment
+    - Metrics
+ 
+#### @aschmahmann
+ - Concluded:
+  - Roadmapping round #1
+  - Basic graph sync + updating working
+ - In progress:
+  - Looking into using existing CRDT tools like y.js to talk to the local Go graph syncing client
+     - Any advice would be appreciated
+ - Blocked:
+  - Not sure how much work to put into improving graph syncing vs wait for IPLD selectors/GraphSync
+  - A little blocked by the lack of the Wallet and Friends components of the Sharing API
```

Comment:
@aschmahmann what is Wallet and Friends? Any url I can read more?

Reply:
Wallet is how you tie a node/device key to an "identity" key. Friends is a way of tracking who you might want to share with. This is still very much in development and hopefully some of it will get cleared up in the Identity Deep Dive call tomorrow. I will likely spend tomorrow afternoon/Wednesday putting the up a proposal for the Sharing API and its sub-components. I'll link back to it here once it's up.

Reply:
The identity call was productive and we are starting to talk about the details. I think the documentation will likely be split between https://github.com/aschmahmann/ipshare and https://github.com/ipfs-shipyard/pm-idm. There's not much there yet, but hopefully more to come next week.